brownie/network/contract.py

```diff
@@ -478,12 +502,12 @@ def _params(abi_params):
     return types
 
 
-def _inputs(abi):
+def _inputs(abi: Any) -> str:
```

Comment:
here and line 510, `abi` is `Dict`.  Guessing this is again something to do with mypy confusion?